src/randoop/main/GenInputsAbstract.java

```diff
@@ -349,53 +305,61 @@ public GenInputsAbstract(String command, String pitch,
   public static boolean no_regression_assertions = false;
 
   /**
-   * The possible values of the regression_assertions_about_exceptions command-line argument.
-   * @see #regression_assertions_about_exceptions
+   * The possible values for exception allocation command-line arguments.
    */
-  public static enum ExceptionAssertionsMode {
-    /** Include no exceptional tests in the regression test suite */
-    NONE,
-    /** Include tests that throw checked exceptions in the regression test suite */
-    CHECKED,
-    /** Include every test in the regression test suite */
-    ALL;
+  public static enum BehaviorType {
+    /** Occurrence of exception should be considered an error */
+    ERROR,
+    /** Occurrence of exception should be considered invalid */
+    INVALID,
+    /** Occurrence of exception should be considered expected behavior */
+    EXPECTED
   }
-
+  
   /**
-   * If a test throws an exception, should it be included in the regression test suite?
+   * Indicates whether a checked exception should be considered as error,
+   * expected, or invalid behavior. Determines how the exception is handled
+   * in generating tests. 
    */
-  @Option("Which exceptional tests to include in the regression test suite")
-  public static ExceptionAssertionsMode regression_assertions_about_exceptions = ExceptionAssertionsMode.ALL;
-
-
-  // TODO make an enum. (But presently Options package requires upper-case
-  // strings for enums, which will require changes to Make targets, plugin, etc.)
-  /** For details, see the Javadoc documentation for {@link randoop.DefaultTestFilter}. */
-  @Option("What kinds of tests to output: pass, fail, or all")
-  public static String output_tests = "all";
+  @Option("Type of behavior assigned to a checked exception")
```

Comment:
I changed these to say 

Whether checked exception is an ERROR, EXPECTED or INVALID

Remember you told me we'd take care of the manual after all this was done.